torch/nn/functional.py

```diff
@@ -1167,12 +1183,15 @@ def linear(input, weight, bias=None):
     """
     if input.dim() == 2 and bias is not None:
         # fused op is marginally faster
-        return torch.addmm(bias, input, weight.t())
-
-    output = input.matmul(weight.t())
-    if bias is not None:
-        output += bias
-    return output
+        _bias = torch.jit._unwrap_optional(bias)
```

Comment:
Nit: maybe don't add a _bias variable and include the _unwrap_optional expr inline ?